README.md

```diff
@@ -32,6 +32,27 @@ This is a work in progress!
 
   * Basic HTTP caching (via `if_modified_since/2` step)
 
+## Installation
+
+You can add Req as a dependency in your mix.exs. Until the first release
+you will have to point to this Git repository.
+
+```elixir
+def deps do
+  [
+    {:req, "~> 0.1.0-dev", github: "wojtekmach/req", branch: "main"}
+  ]
+end
+```
+
+You can also bring Req into your runtime by using `Mix.install/2`:
+
+```elixir
+Mix.install([
+  {:req, "~> 0.1.0-dev", github: "wojtekmach/req", branch: "main"}
+])
+```
```

Comment:
This is kinda redundant, but this package is the easiest way to make requests from iex or Livebook, so perhaps worth mentioning anyway? (especially given that `Mix.install` is new)

Reply:
Good call, I'd even consider the opposite, what if we just mention Mix.install, or that's too much? We could replace the "High-level API" section with something like this:

```markdown
## Usage

The easiest way to use Req is with `Mix.install/2` (requires Elixir v1.12+):

    Mix.install([
      {:req, "~> 0.1.0-dev", github: "wojtekmach/req", branch: "main"}
    ])

    Req.get!("https://api.github.com/repos/elixir-lang/elixir").body["description"]
    #=> "Elixir is a dynamic, functional language designed for building scalable and maintainable applications"

# (maybe we mention how we can add it to mix.exs here after all)

## Low-level API
```


wdyt?

Reply:
Sounds great!

Reply:
fwiw i don't think it would hurt to have mix.exs instructions, but even as a new Elixir dev i guessed i could just copy what's inside the Mix.install into mix.exs deps section.

Reply:
Yeah we hope it is obvious enough but we are happy to revisit it down the road if it is not!